config/routes.rb

```diff
@@ -197,6 +197,7 @@
         end
       end
       resources :levels, except: [:show, :edit, :update]
+      resource :duplication, except: [:index, :edit, :update, :destroy]
```

Comment:
I think you left out `:new`, but I feeling having `only: [:show, :create]` might be easier to parse.